content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,127 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hints
+feature:
+  title: Topology Aware Hints
+  description: >
+    Enables topology aware routing with topology hints on EndpointSlices.
+
+content_type: concept
+weight: 10
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing with topology hints
+included in EndpointSlices. This approach tries to keep traffic close to where
+it originated from. This can result in decreased costs and improved performance.
+
+<!-- body -->
+
+## Introduction
```

Comment:
nit; I'm afraid that following the overview with a section headed “introduction” is a bit of an anti-pattern.
How about “Motivation”?

Reply:
Good catch, also an artifact of a poorly copied page.